src/areaSearch/reducer.js

```diff
@@ -82,6 +84,13 @@ const currentAreaSearchReducer: Reducer<Object | null> = handleActions({
   },
   ['mvj/areaSearch/CREATE_SPECS']: () => null,
   ['mvj/areaSearch/RECEIVE_SPECS_CREATED']: (state, {payload: result}) => result,
+  ['mvj/areaSearch/SET_ATTACHMENTS']: (state, {payload}) => {
```

Comment:
does `payload` have any type?

Reply:
I defined it, and renamed it as attachments, because it does not create an HTTP request but only updates the state locally.